draft-ietf-mops-streaming-opcons.md

```diff
@@ -219,6 +288,46 @@ Presentations:
+
+In recent times, the usage of UDP-based applications that were not simple query-response protocols has grown substantially, and since UDP does not provide any feedback mechanism to senders to help limit impacts on other users, application-level protocols such as RTP {{RFC3550}} have been responsible for the decisions that TCP-based applications have delegated to TCP - what to send, how much to send, and when to send it. So, the way these UDP-based applications interact with other users has changed.
+
+It's also worth pointing out that because UDP has no transport-layer feedback mechanisms, UDP-based applications that expect to send and receive substantial amounts of information must provide their own feedback mechanisms. RTP replies on RTCP Sender and Receiver Reports {{RFC3550}} as its own feedback mechanism, and even includes Circuit Breakers for Unicast RTP Sessions {{RFC8083}} for situations when normal RTP congestion control has not been able to react sufficiently to RTP flows sending at rates that result in sustained packet loss. 
+
+The notion of "Circuit Breakers" has also been applied to other UDP applications, such as tunneling packets that are potentially not congestion-controlled over UDP, in {{RFC8084}}.
+
+## TCP and TCP Protocol Behaviors {#tcp-behavior}
+
+For most of the history of the Internet, we have trusted the TCP protocol to limit the impact of applications that sent a significant number of packets, in either or both directions, on other users. Although early versions of TCP were not particularly good at limiting this impact {{RFC0793}}, the addition of Slow Start and Congestion Avoidance, as described in {{RFC2001}}, were critical in allowing TCP-based applications to "use as much bandwidth as possible, but to avoid using more bandwidth than was possible". Although dozens of RFCs have been written refining TCP decisions about what to send, how much to send, and how when to send it, since 1988 {{Jacobson-Karels}} the signals available for TCP senders remained unchanged - end-to-end acknowledgments for packets that were successfully sent and received, and packet timeouts for packets that were not. 
+
+The success of the largely TCP-based Internet is evidence that the mechanisms used to achieve equilibrium quickly, at a point where TCP senders do not interfere with other TCP senders for sustained periods of time, have been largely successful, even if the specific mechanisms used to reach equilibrium have change over time. Because TCP provides a common tool to avoid contention, new TCP-based applications (for example, HTTP) have had the same transport behavior as older TCP-based applications (for example, FTP). 
```

Comment:
the sentences here are a bit long. Would recommend to break them up a little.

I also personally wouldn't call HTTP a new protocol :D

Reply:
Yeah, I can do that. Coincidentally, I downloaded the diction package yesterday because I was wondering about this. :-)

I can remember when HTTP was a new protocol, but I take your point. The target audience includes people who haven't retired yet :D I'll rephrase this.